packages/postgres-database/src/simcore_postgres_database/models/resource_tracker_licensed_items_checkouts.py

```diff
@@ -22,6 +22,16 @@
         UUID(as_uuid=True),
         nullable=True,
     ),
+    sa.Column(
```

Comment:
Same here as before. Moreover, this way we are sure we do not refer to service key/version pairs